packages/user-mgnt/src/utils/userUtils.ts

```diff
@@ -30,7 +30,8 @@ export const roleScopeMapping: IRoleScopeMapping = {
   PERFORMANCE_OVERSIGHT: ['performance'],
   PERFORMANCE_MANAGEMENT: ['performance'],
   NOTIFICATION_API_USER: ['declare', 'notification-api'],
-  VALIDATOR_API_USER: ['validator-api']
+  VALIDATOR_API_USER: ['validator-api'],
+  CHATBOT_API_USER: ['declare', 'chatbot-api']
```

Comment:
does it need to be on `declare` scope as well ?

Reply:
Yes, it needs either the declare or register scope in order to access the search